### ISSUE TITLE
SCAYT leaks warnings during build process

### ISSUE THREAD

**Issue by mlewand** [ckeditor/ckeditor-docs#165]
It looks like that:

![Warning logs in the console caused by SCAYT](https://user-images.githubusercontent.com/5353898/41850515-225346f2-7885-11e8-8213-74b94e193c9f.png)

It got reported in https://github.com/WebSpellChecker/ckeditor-plugin-scayt/issues/158

We will need a feature to maintain a list of such placeholder values in our docs build.

WSC team provided us with values that are should be used here:

```json
{
    "BRANDING_BASE_SUPPORT_URL": "http://www.webspellchecker.net/support",
    "BRANDING_SUPPORT_MAIL": "support@webspellchecker.net",
    "BRANDING_MIGRATION_MANUAL_URL": "http://wiki.webspellchecker.net/doku.php?id=migration:hosredfreetolicensedck",
    "BRANDING_USER_DICT_MANUAL_URL": "http://wiki.webspellchecker.net/doku.php?id=installationandconfiguration:userdictionaries",
    "BRANDING_CUSTOM_DICT_MANUAL_URL": "http://wiki.webspellchecker.net/doku.php?id=installationandconfiguration:customdictionaries:licensed",
    "BRANDING_MORE_LANG_URL": "http://www.webspellchecker.net/additional-languages-support.html",
    "BRANDING_EC_USER_MANUAL": "https://webspellchecker.net/user_manuals/webspellchecker_proofreader_for_editable_controls_user_manual.pdf",
    "BRANDING_CK_USER_MANUAL": "http://scayt.com/user_manual/scayt_plugin_for_ckeditor4_user_manual.pdf"
}
```

**Comment by mlewand** [comment]
As a temporary viable workaround I have created [SCAYT plugin fork](https://github.com/mlewand/ckeditor-plugin-scayt), which has this values fixed.

**Comment by mlewand** [comment]
Fixed with #170.
